ivy/container/creation.py

```diff
@@ -157,6 +157,26 @@ def full_like(
         dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
         device: Optional[Union[ivy.Device, ivy.NativeDevice]] = None,
     ) -> ivy.Container:
+        """
```

Comment:
You may refer to the docstrings formatting here, please also add docstrings for parameter and return. You can use example types there as reference too!

Reply:
Please reformat for both `full_like` and `static_full_like`

Reply:
parameters description can be taken from functional `full_like` and description from key_chains onwards can be taken from the reference too!